### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Ashish8689**
Change the edit tab and fix some ui issue

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -85,15 +68,15 @@ function App() {
...
-    }
+  const HandleEdit = (id, newinput) => {
+    let updatedtodos = todos.map((todo) => {
+      if (todo.id === id) {
+        todo.value = newinput;
+      }
+      return todo;
+    });
+    settodos(updatedtodos);
+    console.log(updatedtodos);
```
Comment (chirag-madlani) [inline src/App.js]: remove console.log

File: src/App.js [hunk 2]
```diff
@@ -85,15 +68,15 @@ function App() {
...
-    const filterarray = todos.find((todo) => todo.id === id);
-    if (filterarray.status === false) {
-      setinputvalue(filterarray.value);
-      settoggle(true);
-      setedititem(id);
-    } else {
-      alert("Selected Item is Completed");
-    }
+  const HandleEdit = (id, newinput) => {
+    let updatedtodos = todos.map((todo) => {
```
Comment (chirag-madlani) [inline src/App.js]: use const instead of let

File: src/TodoItem.js [hunk 3]
```diff
@@ -9,15 +9,47 @@ function TodoItem({
...
 }) {
+  const [inputDisable, setinputDisabled] = useState(true);
+  const [newinput, setnewinput] = useState(value);
+
+  const HandleSubmit = (e) => {
+    e.preventDefault();
+    HandleEdit(id, newinput);
+    setinputDisabled(true);
+    // setnewinput(value);
+    document.querySelector(".todo-name").style.background = "transparent";
```
Comment (chirag-madlani) [inline src/TodoItem.js]: Don't use this way to update styles, Use state variable & JSX syntax

File: src/TodoItem.js [hunk 4]
```diff
@@ -9,15 +9,47 @@ function TodoItem({
...
   HandleEdit,
 }) {
+  const [inputDisable, setinputDisabled] = useState(true);
+  const [newinput, setnewinput] = useState(value);
+
+  const HandleSubmit = (e) => {
+    e.preventDefault();
+    HandleEdit(id, newinput);
+    setinputDisabled(true);
+    // setnewinput(value);
```
Comment (chirag-madlani) [inline src/TodoItem.js]: Remove commented code

File: src/TodoItem.js [hunk 5]
```diff
@@ -9,15 +9,47 @@ function TodoItem({
...
+  };
+
+  // ***********  After clicking edit button  *************
+  const HandleChangeValue = (e) => {
+    if (status === true) {
+      alert("Item is Completed");
+    } else {
+      setinputDisabled(!inputDisable);
+      setnewinput(value);
+      document.querySelector(".todo-name").style.background = "#dee1ec";
```
Comment (chirag-madlani) [inline src/TodoItem.js]: same as above

File: src/TodoItem.js [hunk 6]
```diff
@@ -9,15 +9,47 @@ function TodoItem({
...
+    HandleEdit(id, newinput);
+    setinputDisabled(true);
+    // setnewinput(value);
+    document.querySelector(".todo-name").style.background = "transparent";
+  };
+
+  // ***********  After clicking edit button  *************
+  const HandleChangeValue = (e) => {
+    if (status === true) {
+      alert("Item is Completed");
```
Comment (chirag-madlani) [inline src/TodoItem.js]: Why are we allowing edit if Item status us complete?

**Review by chirag-madlani — CHANGES_REQUESTED** [review]

**Comment by Ashish8689** [comment]
Changes on all suggestions given,

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**